### PR TITLE
Change constructor binded functions to es6 autobinding functions

### PR DESCRIPTION
**Pull request by somus**
Fixing #158

### CODE REVIEW

File: app/containers/FeaturePage/index.js [hunk 1]
```diff
@@ -17,17 +17,15 @@ import styles from './styles.css';
 class FeaturePage extends React.Component {
   constructor() {
     super();
-    this.onChangeRoute = this.onChangeRoute.bind(this);
-    this.changeRouteToHome = this.changeRouteToHome.bind(this);
   }
 
-  onChangeRoute(url) {
+  onChangeRoute = url => {
```
Comment (mxstbr) [inline app/containers/FeaturePage/index.js]: nit: Brackets around `url`, I.e. Change to `(url)`

File: app/containers/HomePage/index.js [hunk 2]
```diff
@@ -30,19 +30,17 @@ import styles from './styles.css';
...
   constructor() {
     super();
-    this.onChangeRoute = this.onChangeRoute.bind(this);
-    this.changeRouteToReadme = this.changeRouteToReadme.bind(this);
   }
 
   shouldComponentUpdate = shouldPureComponentUpdate;
 
-  onChangeRoute(url) {
+  onChangeRoute = url => {
```
Comment (mxstbr) [inline app/containers/HomePage/index.js]: nit: Brackets around `url`

**Comment by mxstbr** [comment]
Wow great work, thanks so much! Two small nitpicks then I'll merge!

File: app/containers/FeaturePage/index.js (hunk 1 above)
Comment (somus) [inline app/containers/FeaturePage/index.js]: Brackets can be skipped when there is only one parameter. It is compulsory when there is either zero or more than one parameter.

Comment (mxstbr) [inline app/containers/FeaturePage/index.js]: I know, it's nice to have them to explain intent and maybe somebody wants to add multiple params later on.

**Comment by mxstbr** [comment]
Thanks so much!

**Pull request merged**